### DISCUSSION TITLE
Uninstall Script for Ubuntu

### DISCUSSION THREAD

**Discussion by clairmont32** [Security-Onion-Solutions/securityonion#5769] [Ideas] [answered]
Greetings,

I didnt see an uninstall mechanism in the docs for Ubuntu and I'd like to remove it from my NUC to keep it in a leaner state for other needs that come along. What's the recommended way to do so?

**Comment by dougburks** [comment, marked as answer]
The quickest and easiest option right now is to perform a fresh OS installation.
